data/homepage.yaml

```diff
@@ -87,6 +87,22 @@ blog:
   title: Blogs
   intro: Read all about it!
   url: /blog/
+contribute:
+  title: Contribute
+  ways:
```

Comment:
These should probably just be a list rather than hardcoded names. But let's not block this PR, I will see about a second patch that uses an iterator for it.

Reply:
Oh, you did use an iterator, silly me. Sending a minor patch to this file to simplify....

Reply:
Thanks for improving/simplifying it though @GregSutcliffe I was going the long way around and making things overly complex.